sdk/monitor/Azure.Monitory.Query/tests/LogsQueryClientClientLiveTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.Monitory.Query;
+using Azure.Monitory.Query.Models;
+using NUnit.Framework;
+
+namespace Azure.Template.Tests
+{
+    public class LogsQueryClientClientLiveTests: RecordedTestBase<MonitorQueryClientTestEnvironment>
+    {
+        public LogsQueryClientClientLiveTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        private LogsClient CreateClient()
+        {
+            return InstrumentClient(new LogsClient(
+                TestEnvironment.Credential,
+                InstrumentClientOptions(new LogsClientOptions())
+            ));
+        }
+
+        [Test]
+        public async Task CanQuery()
+        {
+            var client = CreateClient();
+
+            var results = await client.QueryAsync(TestEnvironment.WorkspaceId, "Heartbeat");
+
+            var resultTable = results.Value.Tables.Single();
+            CollectionAssert.IsNotEmpty(resultTable.Columns);
```

Comment:
validate that we returned the expected record?

Reply:
Unfortunately there is no data in that table. I'm working on how to get the test data into the service to write better tests.